src/pretix/control/views/item.py

```diff
@@ -820,7 +821,9 @@ def get_queryset(self):
         }
 
         if self.request.GET.get("ordering", "-date") in valid_orders:
-            qs = qs.order_by(*valid_orders[self.request.GET.get("ordering", "-date")])
+            qs = qs.order_by(*get_deterministic_ordering(Quota, valid_orders[self.request.GET.get("ordering", "-date")]))
```

Comment:
Should we instead of using `get_deterministic_ordering` just define the proper order-tuples in `valid_orders` just above this code section? I.e.:

```
        valid_orders = {
            '-date': ('-subevent__date_from', 'name', 'pk'),
            'date': ('subevent__date_from', '-name', 'pk'),
            'size': ('size', 'name', 'pk'),
            '-size': ('-size', '-name', 'pk'),
            'name': ('name', 'pk'),
            '-name': ('-name', 'pk'),
        }
```